backend/src/main/scala/com/softwaremill/bootzooka/http/model/headers/headers.scala

```diff
@@ -0,0 +1,86 @@
+package com.softwaremill.bootzooka.http.model.headers
```

Comment:
I suspect the package mimicks what's in akka-http, but maybe it would make more sense to move it unider the `api` package as e.g. `owasHeaders.scala`?